tests/test_bursts.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+from s1reader.s1_burst_slc import Sentinel1BurstSlc
+
+def test_burst(bursts):
+    for i, burst in enumerate(bursts):
+        expected_burst_id = f't71_iw3_b{844 + i}'
+        assert burst.burst_id == expected_burst_id
```

Comment:
Thanks @LiangJYu . This is great start. I think we should go ahead and start checking all different burst data members for the specific burst of interest. You need to simply lookup the xml file manually and compare the extracted value by the reader with what you expected. 

Let's start with scalars and simpler parameters. Then let's discuss for more complicated parameters or methods (like azimuth_carrier etc).